mubench.reviewsite/templates/includes/review_modal.phtml

```diff
@@ -0,0 +1,76 @@
+    <div class="modal-content">
+                            <span class="close"
+                                  id="<?= htmlspecialchars($project) ?>_<?= htmlspecialchars($version) ?>_<?= htmlspecialchars($misuse->id) ?>">&times;</span>
+        <table class="invisible">
+            <tr><td><b>Detector:</b></td><td><?= htmlspecialchars($detector->muid); ?></td></tr>
+            <tr><td><b>Target:</b></td><td>project '<?= $markdown_parser->text($misuse->getProject()); ?>' version <?= $markdown_parser->text($misuse->getVersion()); ?></td></tr>
+            <tr><td><b>Misuse:</b></td><td>misuse '<?= $markdown_parser->text($misuse->misuse_muid); ?>'</td></tr>
+            <tr><td><b>Tags:</b></td>
+                <td>
+                    <?php foreach($misuse->misuse_tags as $tag) : ?>
+                        <div class="misuse-tag">
+                            <span><?= htmlspecialchars($tag->name) ?></span>
+                        </div>
+                    <?php endforeach; ?>
+                </td>
+            </tr>
+        </table>
+        <h2>Reviews:</h2>
+        <table class="invisible">
+            <tr>
+            <?php foreach($misuse->reviews as $review) : ?>
+            <td>
+                <div class="card-shadow">
+                <table class="invisible">
+                    <tr>
+                        <td><b>Reviewer:</b></td>
+                            <td><?= htmlspecialchars($review->reviewer->name) ?></td>
+                    </tr>
+                    <tr style="padding-top:0;margin-top:0;">
+                        <td><b>Decision:</b></td>
+                                <td>
+                                    <?php if($review->identifiesHit()) : ?>
+                                        <i class="fa fa-thumbs-o-up" aria-hidden="true" style="color:green;" title="Yes"></i>
+                                        <?php
+                                        $number_in_circle = $review->getLowestHitRank();
+                                        $circle_color = "black";
+                                        include "number_circle.phtml";
+                                        ?>
+                                    <?php elseif($review->getDecision() == \MuBench\ReviewSite\Models\Decision::NO) : ?>
+                                        <i class="fa fa-thumbs-o-down" aria-hidden="true" style="color:darkred;" title="No"></i>
+                                    <?php else : ?>
+                                        <i class="fa fa-question-circle" aria-hidden="true" style="color:blue;" title="Needs Clarification"></i>
+                                    <?php endif; ?>
+                                </td>
+                    </tr>
+                    <tr>
+                        <td><b>Comment:</b></td>
+
+                    </tr>
+                    <tr><td colspan="2">
+                            <?= $markdown_parser->text($review->comment); ?>
```

Comment:
This somehow does not convert newlines into `<br/>`.

Reply:
this is standard markdown behavior, only newlines in paragraphs are converted.

Reply:
I can't seem to get a line break at all (be it through `<br>` or new `<p>`). How do I start a new paragraph? Blank line doesn't seem to be working. What's the "standard behavior" for this?

Reply:
This is an example how it should give you a line break:

"## par 1
This is a line.  
And this is after the break."

I also just found an option to enable normal line breaks.

Reply:
Isn't `##` syntax for headers? Anyways, this is certainly too verbose just for a simple line break in a review comment.

Just to make sure we're on the same page: I don't need linebreaks in the form of `<br>`. I want to have *an* intuitive way to get some text to start in a new line. This may as well be through starting a new paragraph. However, this, as well, does not seem to work as expected (through inserting a blank line).

Can you provide a page summarizing the Markdown variant we're using?

Reply:
I couldn't really find a page summarizing it. I tested around a little and apparently when you end a line with 2 spaces and then you start a new line it inserts a linebreak.

Reply:
* [ ] create markdown parser once, globally
* [ ] enable line breaks on parser